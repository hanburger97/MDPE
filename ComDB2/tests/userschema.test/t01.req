set user mohit
set password mohit
put authentication on
select * from comdb2_users
