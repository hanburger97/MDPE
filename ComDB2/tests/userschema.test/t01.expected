(username='default', isOP='N')
(username='mohit', isOP='Y')
